build-logic/common-plugins/src/main/kotlin/DebugBuild.kt

```diff
@@ -97,24 +97,22 @@ fun Project.printResolvedConfigFiles(level: LogLevel = LIFECYCLE, configName: St
     val files = config.resolvedConfiguration.resolvedArtifacts.map { it.file }
     logger.log(level, "$prefix Configuration '$configName' has ${files.size} files:")
     files.forEach {
-        logger.log(level, "$prefix    file: '$it'")
+        logger.log(level, "$prefix file: '$it'")
     }
 }
 
 fun Project.printTaskInputs(level: LogLevel = LIFECYCLE, taskName: String) {
     val task = tasks.getByName(taskName)
     val inputs = task.inputs.files
     logger.log(level, "$prefix Task '$taskName' has ${inputs.count()} inputs:")
-    inputs.forEach { logger.log(level, "$prefix     input: '$it' (type: ${it.javaClass.name})") }
-    //inputs.asFileTree.forEach { logger.log(level, "$prefix   input : '$it'") }
```

Comment:
Why did you remove this code? It's useful in that a directory doesn't tell you a lot. If it has files (which it will when resolved, hence printTaskInputs should only be called from the execution phase), it's quite useful to see what files it contains. However, it should probably be on a more fine grained log level, but I haven't implemented that.

Reply:
I didn't remove the code, only the commented out line, which seemed to be a trivial variation on the un-commented one. The diff here is misleading - please take a look at the file itself